src/Sinch/Verification/VerificationMethod.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Sinch.Core;
+
+namespace Sinch.Verification
+{
+    /// <summary>
+    ///     The method of the verification request.
+    /// </summary>
+    /// <param name="Value"></param>
+    public record VerificationMethod(string Value) : EnumRecord(Value)
```

Comment:
Maybe this record can be renamed to `VerificationStatusMethod` to match the documentation

Reply:
Up to you, but as it is also described as `method` in the Verifications API, I was thinking about mentioning the VerificationStatus API name to clearly make the distinction.